### ISSUE TITLE
PouchDB fetch failed, while using ionic webview

### ISSUE THREAD

**Issue by Sudalaiyandi** [pouchdb/pouchdb#7546]
### Issue
I have developed ionic application with ionic webview plugin, from that application i’ve navigate to iOS native page using broadcast plugin.I need data for iOS page from pouchdb. My scenario, while login my ionic application it hits our server to get one ionic application then write it on document path(which is in iOS application folder).The pages of ionic, should be shown from downloaded ionic application and pouchdb fetch code also available.Make request from iOS native page and call pouchdb fetch method using broadcast plugin, but without showing any error and response from pouchdb and nothing happened.

But important thing, if i uninstall ionic webview plugin, pouch db fetch should be done properly.While i make request from native page.It seems ionic webview plugin did some impact on pouchdb fetching.

Kindly provide solution for this case.

### Info
- Environment: hybrid app
- Platform: iOS
- Adapter: IndexedDB
- Server: CouchDB

### Reproduce
Make request from iOS native page, then call pouch db fetch method in ionic page with
webview plugin.

**Comment by karlwestin** [comment]
Sounds like you're having an awesome project going!

I'm sorry but your description is a little bit hard to follow, it also seems to be very specific to the application you're building.

If you can figure out the ionic parts, narrow down your question to involve the pouchdb specific things only, so we can look at the issue in isolation, we can provide further help

Thank you, good luck :)

**Comment by Sudalaiyandi** [comment]
From iOS native page, i can make request to ionic page using broadcast plugin like this
`this.db.rel.find("student", "3c239147476cb4a0524a1f175002401c").then(res => {
          console.log(res);
        }).catch(error =>{
          console.log(error);
        })`
But rel.find is not working, without showing any error and response from pouchdb and nothing happened.
But important thing, if i uninstall ionic webview plugin, pouch db fetch should be done properly and got response.

**Comment by karlwestin** [comment]
ok im trying to set expectations straight here
you're working on a commercial application, asking people to take time off to help

You are asking us to try to reverse engineer a very complicated ionic (not pouchdb) setup, using a broadcast plugin, which seems to be for some kind of commercial project since you can not link us to the code

Please either 
- set up a repository where someone can build an application which shows very clearly what the issue is
- pay someone to help you on your commercial project

thank you

**Comment by Sudalaiyandi** [comment]
@karlwestin I can understand your point. If i uninstall ionic webview plugin, PouchDB fetch is worked properly.Is there any link with PouchDb and ionic webview plugin.

Thanks

**Comment by daleharvey** [comment]
So I think there may be a little lost in translation here, just for next time @Sudalaiyandi 

> Kindly provide solution for this case.

At least to me and possibly others could be considered more of a demand than a request and in an OSS it is fairly rude to demand things. 

As for the problem, there is no specific link between ionic and pouchdb, if ionic is breaking requests made from pouchdb when it is installed then it is most certainly a but with ionic, occasionally we can workaround these things but it would need you to investigate what the actual problem is and propose a solution. Since this looks to be a bug in ionic closing out. If you think there is an issue in pouchdb if you could investigate what the issue is and at the least provide us clear steps to reproduce so we can see the issue for ourselves, cheers

**Comment by Sudalaiyandi** [comment]
@daleharvey I can identified data fetch is not blocked by ionic web view.I used pouchdb adpater as 
> pouchdb-adapter-cordova-sqlite 2.0.3

I used `cordova-plugin-sqlite-2` for local database access.If I used this plugin index not created and find api not working. But i uninstalled  `cordova-plugin-sqlite-2` plugin data fetch is worked from ionic to ios native.Provide solution for my issue.
